lib/tasks/revival.thor

```diff
@@ -29,7 +29,6 @@ class Revival < Thor
   SERVICE_ASSOCIATIONS = [
     { service_plans: [:service_contracts] },
     :application_plans,
-    { end_user_plans: [:plan_limits]},
```

Comment:
I guess this is correct but what do we use this revival.thor for? @3scale/system

Reply:
I made it ... This is to revive some deleted data from a database backup. I did few times to restore some accidental removal of tenants :D 
I think it is outdated now anyway